tool/grind.dart

```diff
@@ -50,7 +50,6 @@ serveLocalAppEngine() async {
   ConstTaskArgs('build', flags: {
     _debugFlag: true,
   }, options: {
-    _preNullSafetyServerUrlOption: 'http://127.0.0.1:8082/',
     _nullSafetyServerUrlOption: 'http://127.0.0.1:8084/',
```

Comment:
Don't be afraid to rename `_nullSafetyServerUrlOption` (or related flags) if that name no longer makes sense in an NNBD-only world.

Reply:
Done